docs/static/breaking-changes.asciidoc

```diff
@@ -29,7 +30,155 @@ and have the persistent queue enabled, we strongly recommend that you drain or
+
+[float]
+===== Codec Plugins
+
+*CEF Codec*
+
+* Removed obsolete `sev` option
+* Removed obsolete `deprecated_v1_fields` option
+
+*Netflow Codec*
+
+* Changed decoding of application_id to implement RFC6759; the format changes from a pair of colon-separated ids (e.g. `0:40567`) to a variable number of double-dot-separated ids (e.g. `0..12356..40567`).
+
+[float]
+===== Filter Plugins
+
+*Clone Filter*
+
+* Make `clones` a required option
+
+*Geoip Filter*
+
+* Removed obsolete `lru_cache_size` option
+
+*HTTP Filter*
+
+* Removed obsolete `ssl_certificate_verify` option
+
+[float]
+===== Input Plugins
+
+*Beats Input*
+
+* Removed obsolete `congestion_threshold` option
+* Removed obsolete `target_field_for_codec` option
+* Changed default value of `add_hostname` to false
+
+NOTE: In Beats 7.0.0, the fields exported by Beats _to_ the Logstash Beats Input
+conform to the {ecs-ref}/index.html[Elastic Common Schema (ECS)]. Many of the
+exported fields have been renamed, so you may need to modify your pipeline
+configurations to access them at their new locations prior to upgrading your
+Beats. See {beats-ref}/breaking-changes-7.0.html[Beats Breaking changes in 7.0]
+for the full list of changed names.
+
+*HTTP Input*
+
+* Removed obsolete `ssl_certificate_verify` option
+
+*HTTP Poller Input*
+
+* Removed obsolete `interval` option
+* Removed obsolete `ssl_certificate_verify` option
+
+*Tcp Input*
+
+* Removed obsolete `data_timeout` option
+* Removed obsolete `ssl_cacert` option
+
+[float]
+===== Output Plugins
+
+*Elasticsearch Output*
+
+* Remove support for parent/child (still support join data type) since we don't support multiple document types any more
+* Removed obsolete `flush_size` option
+* Removed obsolete `idle_flush_time` option
+
+*HTTP Output*
+
+* Removed obsolete `ssl_certificate_verify` option
+
+*Kafka Output*
+
+* Removed obsolete `block_on_buffer_full` option
+* Removed obsolete `ssl` option
+* Removed obsolete `timeout_ms` option
+
+*Redis Output*
+
+* Removed obsolete `queue` option
+* Removed obsolete `name` option
+
+*Sqs Output*
+
+* Removed obsolete `batch` option
+* Removed obsolete `batch_timeout` option
+
+*Tcp Output*
+
+* Removed obsolete `message_format` option
+
 [float]
+===== Beats input
+
+Beats fields have been renamed to map to Elastic Common Schema (ECS).
+
+Starting with 7.0, the fields exported by {beats} conform to the
+{ecs-ref}/index.html[Elastic Common Schema (ECS)]. Many of the exported fields
+have been renamed. See {beats-ref}/breaking-changes-7.0.html[Beats breaking
+changes in 7.0] for the full list of changed names.
+
+See the *{beats} Platform Reference* for more information on
+{beats-ref}/upgrading-6-to-7.html#enable-ecs-compatibility[Beats and ECS].
+
+TODO:  What action does user need to take?  
+
+[float]
+===== Elasticsearch output
+
+ILM on by default
```

Comment:
@yaauie can we add here that in 7.0 ILM will be auto detected and enabled by default if the ES cluster supports it?

Reply:
Here's a peek at what's coming in my next commit: 
```
* {es} {ref}/index-lifecycle-management.html[Index lifecycle management (ILM)] is
auto-detected and enabled by default if your {es} cluster supports it.
```
Comments are welcomed, pre- or post-commit.